libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -649,11 +649,13 @@ void RC_Channel::do_aux_function_fence(const aux_switch_pos_t ch_flag)
     AP_Logger *logger = AP_Logger::get_singleton();
     if (ch_flag == HIGH) {
         fence->enable(true);
+        gcs().send_text(MAV_SEVERITY_INFO, "Fence enabled");
```

Comment:
This would more properly go into the enable method on the fence.  The WriteEvent too, probably.

Reply:
give me a pointer to the areas, since I am less than competent in this, and I will modify...@rmackay9 has no objections to confirmations of user actions, re option rc switches...just to needless spam...its very helpfull when running a TX gcs like Yaapu to have feedback on some things like this with many switches on the TX...

Reply:
https://github.com/ArduPilot/ardupilot/pull/14489/files